http.md

```diff
@@ -0,0 +1,340 @@
+# JSON RPC HTTP Methods
+
+The block engine accept HTTP requests using the [JSON-RPC 2.0](https://www.jsonrpc.org/specification) specification.
+
+## RPC HTTP Endpoint
+
+**Default port:** 443 e.g. [http://localhost:443/bundles](http://localhost:443/bundles), [http://192.168.1.88:443/bundles](http://192.168.1.88:443/bundles)
+
+## Request Formatting
+
+To make a JSON-RPC request, send an HTTP POST request with a `Content-Type: application/json` header. The JSON request data should contain 4 fields:
+We follow the same request formatting as Solana. You can find the documentation [at](https://docs.solana.com/api/http#request-formatting)
+
+## Definitions
+
+- Bundle: Bundles are a list of transactions that execute sequentially and atomically. “All or nothing” so to speak. This means that a user can send a bundle that contains multiple transactions and guarantee that they are all executed one after the other and the bundle succeeds only if all individual transactions succeed.
+- Hash: A SHA-256 hash of a chunk of data.
+- Pubkey: The public key of a Ed25519 key-pair.
+- Tip Account: List of accounts to which a tip can be paid for processing the bundles. The searcher/user submitting the bundle must pay the tip for the bundle processing.
+- Transaction: A list of Solana instructions signed by a client keypair to authorize those actions.
+- Signature: An Ed25519 signature of transaction's payload data including instructions. This can be used to identify transactions.
+
+## Configuring State Commitment
+
+For preflight checks and transaction processing, Solana nodes choose which bank
```

Comment:
I don't think we'll be doing any preflight checks

Reply:
Removed it to avoid confusion. 
Now reads,

```
## Configuring State Commitment

Solana nodes choose which bank state to query based on a commitment requirement set by the client.
The commitment describes how finalized a block is at that point in time. When
querying the ledger state, it's recommended to use lower levels of commitment
to report progress and higher levels to ensure the state will not be rolled back.
```